src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+/**
+ * @title Interface for a contract the coordinates between various registries for an AVS.
+ * @author Layr Labs, Inc.
+ */
+interface IRegistryCoordinator {
```

Comment:
I still prefer the 'subregistry' language, but perhaps can be settled later

Reply:
got it